icu4j/main/classes/core/src/com/ibm/icu/util/MeasureUnit.java

```diff
@@ -567,7 +568,7 @@ public MeasureUnit product(MeasureUnit other) {
      * If this is a SINGLE unit, a list of length 1 will be returned.
      *
      * @return An unmodifiable list of single units
-     * @internal ICU 68 Technology Preview
+     * @draft ICU 68 Technology Preview
```

Comment:
@hugovdm @sffc 

This should be `@draft`, right ?

Reply:
Looks like it yes: I find no other provisional & internal methods, I find plenty of provisional & draft methods. Thanks!

Reply:
@draft should not come with Technology Preview. Please just use `@draft ICU 68`. Thanks.

Reply:
done